lib/ansible/modules/network/ios/ios_vlan.py

```diff
@@ -211,38 +210,52 @@ def map_params_to_obj(module):
     return obj
 
 
-def map_config_to_obj(module):
```

Comment:
Also, I would probably want you to add some unit tests for the functions, which we introduced here as well as `map_config_to_obj(module)`, which we are changing in this PR.